app/src/main/java/com/glowstudio/android/blindsjn/feature/community/WritePostScreen.kt

```
@@ -0,0 +1,250 @@
+
+    val context = LocalContext.current
+    val scope = rememberCoroutineScope()
+    val repository = remember { CommunityRepository() }
+    val businessRepository = remember { BusinessCertRepository() }
+
+    // 업종 목록 로드
+    var industries by remember { mutableStateOf<List<Industry>>(emptyList()) }
+    val userId by UserManager.getUserId(context).collectAsState(initial = -1)
+    val phoneNumber = UserManager.getPhoneNumber(context)
+
+    LaunchedEffect(Unit) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)